.storybook/Icon.js

```diff
@@ -8,20 +8,11 @@ const oldKeys = ['srollLeft', 'chevronLight', 'chevronThick']
 
 storiesOf('Icon', module)
   .add('Index', () => (
-    <Box
-      p={2}
-      color='white'
-      bg='blue'>
+    <Box p={2} color="white" bg="blue">
       <Flex wrap>
         {keys.map(name => (
-          <Box
-            key={name}
```

Comment:
(n-b) I kinda prefer them on separate lines

Reply:
Or did prettier do this lol?

Reply:
Yeah, I believe prettier is taking control here, so no more need to comment on code style :)